woocommerce/wc-admin
# Set Node version to 14 in .nvmrc to prevent incompatible versions of Node from being used with nvm

Recently, the latest LTS version of Node was updated to 16. Node 16 comes with npm 8. The `package-lock.json` file in `woocommerce-admin` does not work with npm 7+.

This PR explicitly sets the Node version to 14 in `.nvmrc`, as we know that version is supported (Node 14 comes with npm 6). Once we fix #7701 to support Node 16/npm 8, we should update `.nvmrc` to set the Node version to 16.

Partially addresses #7701. 

### Detailed test instructions:

1. Check out this branch
2. Run `nvm use` and confirm that Node 14 and npm 6 are used.
3. Run `npm install` and confirm that it succeeds (no errors such as described in #7701)
